scripts/install

```diff
@@ -73,7 +73,19 @@ ynh_exec_warn_less yunohost firewall allow --no-upnp UDP $port_wg
 #=================================================
 ynh_script_progression --message="Installing dependencies..." --time --weight=1
 
-ynh_install_extra_app_dependencies --repo="http://deb.debian.org/debian buster-backports main" --package="$pkg_dependencies"
+#Add buster-backports repo
+ynh_add_repo --uri="http://deb.debian.org/debian" --suite="buster-backports" --component="main" --name="$app"
+
+#Add pin-priority for wireguard package
+ynh_pin_repo --package="wireguard*" --pin="origin deb http://deb.debian.org/debian buster-backports main" 995 --name="$app"
```

Comment:
Can you explain why it is needed to "explode" the `ynh_install_extra_app_dependencies` helper? If I read its code right, there is already a 995 priority set on the repositories added by it.

Reply:
I want to pin-priority only wireguard packages, If I don't it will update linux-image-amd64 to the buster-backports version and install the new kernel

Reply:
When you install wireguard_ynh, the installation of wireguard provoke the update of linux-image-amd64 to the buster-backports version installing the new kernel in the process.
And if you add wireguard-dkms, it will install the linux-header of the buster-backports kernel because of the new linux-image-amd64.
The solution was to set the pin-priority to 995 for only wireguard packages, the only way to set this is to "explode" this herlper function to change ```--package="*"``` to ```--package='wireguard*'```

Reply:
By the way I think I should add a way to uninstall the buster-backports linux-image-$ARCH by the right one in the upgrade script